app/pods/cases/case/subcases/overview/controller.js

```diff
@@ -1,18 +1,22 @@
 import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency-decorators';
 
-export default Controller.extend({
-  queryParams: ['refresh'],
 
-  isShowingOptions: false,
-  isShowingProcess: false,
-  refresh: false,
+export default class CaseSubcasesOverviewController extends Controller {
+  @service toaster;
+  @service intl;
 
-  actions: {
-    backLinkAction() {
-      this.transitionToRoute('cases');
-    },
-    refresh() {
-      this.toggleProperty('refresh');
-    },
-  },
-});
+  @task
+  *saveSubcase(subcase) {
+    yield subcase.save();
+    this.toaster.success(this.intl.t('successfully-saved'));
```

Comment:
Volgens de nieuwe "toast-policy" is deze succes toast niet nodig aangezien gebruikers expliciet op een opslaan-button klikken en het geen autosave is. Zelfde bij `saveCase`.